package.json

```diff
@@ -14,9 +14,10 @@
     "lint:prettier": "prettier './**/*.js' './**/*.css' './**/*.md' --list-different",
     "typecheck": "tsc",
     "build": "yarn run clean && yarn run rollup -c",
+    "checkimport": "grep 'import [^*{]' dist/react-stripe.d.ts; case $? in 1) true ;; 0) echo Found invalid import; false ;; *) false ;; esac",
```

Comment:
(2) Neat. Would be nice to read this in a separate file in a `scripts` directory, with a comment on how to fix this issue

Reply:
Done.